### PR TITLE
v3.1

### PR DESCRIPTION
**Pull request by Rexogamer**
# New stuff
**feat:** ability to pay others during calls
**internals**: split aliases into a different file

### CODE REVIEW

**Comment by Mitchell3514** [comment]
Quality could be better

**Comment by Rexogamer** [comment]
There’s not a tab button on mobile :/ although the issue about aliases not being defined confuses me

as for CLA be quiet a bot can’t sign

File: Commands/Public/pay.js [hunk 1]
```diff
@@ -112,15 +112,15 @@ module.exports = async(client, msg, suffix) => {
...
+	client.log(`💸 User ${msg.author.tag} (${msg.author.id}) gave ¥${amount} to ${user.tag} (${user.id})`);
 	user.send({ embed: {
 		color: config.colors.receipt,
 		author: {
 			name: `${msg.author.tag} (${msg.author.id})`,
 			icon_url: msg.author.displayAvatarURL(),
 		},
 		title: "You received money!",
-		description: "Someone has transferred money to you. Here's your receipt.",
+		description: "Someone has given money to you. Here's your receipt.",
```
Comment (Mitchell3514) [inline Commands/Public/pay.js]: 'someone has given you money'

Comment (Rexogamer) [reply Commands/Public/pay.js]: updated wording

**Pull request merged**